docs/book/src/topics/machinepools.md

```diff
@@ -91,8 +91,6 @@ spec:
   awsLaunchTemplate:
     instanceType: "${AWS_CONTROL_PLANE_MACHINE_TYPE}"
     sshKeyName: "${AWS_SSH_KEY_NAME}"
-  subnets:
```

Comment:
We can keep this in the docs:
subnets:
  - subnet-id

Btw, did you check if this field is working by any chance?

Reply:
Yes will receive a validation error. It says ` invalid type for io.x-k8s.cluster.infrastructure.v1beta1.AWSMachinePool.spec.subnets: got "string", expected "map";`

So that's why I removed it from doc as well. Please let me know if you think we should still keep it?